org.opentosca.yamlconverter.yamlmodel/src/org/opentosca/yamlconverter/yamlmodel/yaml/element/Input.java

```diff
@@ -7,7 +7,7 @@
 public class Input extends YAMLElement {
 
 	private String type = "";
-	private String defaultValue = "%%USER_INPUT%%";
+	private String defaultValue = "%%USER_INPUT_REQUIRED%%";
```

Comment:
I think we should not set a default value by default here! The problem with that is, that if we set a default default value, it's more difficult to find out, whether the input definition defined the default or our parser. This is important to know, because if there is no default value in the input, we should first look for the default value in the NodeType rather than using the global default.

Ranking:
1) Userinput
2) Input-default
3) Node-/Relationship-/Whatever-Type-default
4) Global default